package.json

```diff
@@ -202,12 +202,13 @@
     "@hapi/call": "^8.0.1",
     "@hapi/subtext": "^7.0.3",
     "@kubernetes/client-node": "^0.16.3",
+    "@ogre-tools/fp": "5.2.0",
```

Comment:
How this related to PR?

Reply:
It isn't but I think @jansav added this value manually (instead of using `npm i --save` or `yarn add` and so added it in the wrong place. Both of those CLIs will sort these two objects whenever they touch them.